src/Collection/Collection.router.ts

```diff
@@ -146,4 +154,10 @@ export class CollectionRouter extends Router {
     ])
     return true
   }
+
+  async isCollectionNameAvailable(collection: CollectionAttributes) {
+    const dbCollection = await Collection.findByName(collection.name.trim())
+
+    return dbCollection.length === 0
```

Comment:
If this is the only use we're going to have for `findByName`, you can always speed things up by adding a `Collection.nameExists` which uses a count